src/fgimic/jfield.f90

```diff
@@ -522,6 +527,101 @@ subroutine acid_vtkplot(this)
         call write_vtk_imagedata('acid.vti', this%grid, val)
         deallocate(val)
     end subroutine
+
+    subroutine get_property(this)
+        ! assume external grid is read in correctly via read grid 
+        ! assume grid_w.grd, coord.au, gridfile.grd are in the same directory
+        type(jfield_t) :: this
+        real(DP), dimension(:,:), pointer :: jtens
+        real(DP), allocatable :: wg(:), coord(:,:), grd(:,:) 
+        real(DP), dimension(3) :: d, bb, jvec, sigma, chi
+        real(DP) :: f, tmp
+        integer :: i, j, k, npts, natoms
+
+        jtens => this%tens
+
+        ! get atom coordinates in bohr
+        open(unit=15, file="coord.au")
+        natoms = getnlines(15)
+        allocate(coord(natoms,3))
+        do i=1, natoms
+          read(15,*) coord(i,1:3)
+        end do
+        close(15)
+
+        ! get weights from numgrid files (external)
+        open(unit=15, file="grid_w.grd")
+        open(unit=16, file="gridfile.grd")
```

Comment:
should we not put both into the same file instead of two files?

Reply:
No. At the moment gimic reads in the grid file as xyz information. We can do this later on. The ordering is important to make sure that the J tensor is assigned correctly.

Reply:
I think one file is really better than two. But I understand if you don't want to change it, then I can change it later.

Reply:
I completely agree but we can adapt this later on. At the moment it would only complicate things. I tried to touch as little as possible inside gimic since we are after a completely different python oriented solution.

Reply:
OK - convinced :-)